src/ida/ida_io.c

```diff
@@ -1418,16 +1418,16 @@ int IDAPrintAllStats(void* ida_mem, FILE* outfile, SUNOutputFormat fmt)
   {
   case SUN_OUTPUTFORMAT_TABLE:
     /* step and method stats */
-    fprintf(outfile, "Current time                 = %" RSYM "\n",
+    fprintf(outfile, "Current time                 = " SUN_REAL_FORMAT_G "\n",
```

Comment:
Update this `PrintAllStats` function to use the `sunfprintf_*` functions like in ARKODE and CVODE